atom/common/asar/archive.cc

```diff
@@ -107,6 +107,8 @@ bool FillFileInfoWithNode(Archive::FileInfo* info,
     return false;
   info->offset += header_size;
 
+  node->GetBoolean("executable", &info->executable);
```

Comment:
In C++ the value of a variable is undefined until you explicitly initialize it, and `node->GetBoolean` only changes `info->executable` when the `executable` is specified, so `info->executable` could end up with arbitrary value.